java_src/foam/dao/index/AltIndex.java

```diff
@@ -0,0 +1,108 @@
+/**
```

Comment:
User the shorter header:

/**
 * @license
 * Copyright 2017 The FOAM Authors. All Rights Reserved.
 * http://www.apache.org/licenses/LICENSE-2.0
 */